src/cli/Utils.h

```diff
@@ -40,7 +40,7 @@ namespace Utils
     extern FILE* DEVNULL;
 
     void setStdinEcho(bool enable);
-    QString getPassword(FILE* outputDescriptor = STDOUT);
+    QString getPassword(FILE* errorDescriptor = STDERR);
```

Comment:
I would prefer keeping this variable named `outputDescriptor` despite the default being `STDERR`. It's more generic and less confusing as to what this does.